explain-views.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Views
+summary: Learn about the execution plan information returned by the `EXPLAIN` statement in TiDB.
+---
+
+# Views
+
+`EXPLAIN` will display the tables and indexes that a [view](/views.md) references, and not the name of the view itself. This is because views are only virtual tables and don't store any data themselves. The definition of the view, and the rest of the statement are merged together during SQL optimization.
```

Comment:
```suggestion
`EXPLAIN` displays the tables and indexes that a [view](/views.md) references, not the name of the view itself. This is because views are only virtual tables and do not store any data themselves. The definition of the view and the rest of the statement are merged together during SQL optimization.
```